uncompressed/jitsi/index.js

```diff
@@ -1,2 +1,6 @@
 "use strict";
-module.exports = Ferdi => Ferdi;
\ No newline at end of file
+module.exports = Franz => class Jitsi extends Franz {
+  overrideUserAgent() {
+    return window.navigator.userAgent.replace("Electron ","Electron/");
```

Comment:
Can you check if the following user agent works? This is what has been followed in most of the other recipes and it would be great if it works so that we can maintain uniformity among recipes. 
(Reference: #115)

```suggestion
    return window.navigator.userAgent.replace(/(Ferdi|Electron)\/\S+ \([^)]+\)/g, '').trim();
```

Reply:
Hello,
It works but the detected browser is Chrome and not Electron, I don't know if it will create some issues after.

Reply:
@tristanplouz: That shouldn't be a problem. Whatsapp uses the same code and the user-agent that is being sent from my test machine is 
```
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36 Ferdi/5.5.0 (Electron 8.2.3)
```
Just trying to maintain uniformity here.